config/initializers/assets.rb

```diff
@@ -2,6 +2,10 @@
 #   licensed under the Affero General Public License version 3 or later.  See
 #   the COPYRIGHT file.
 
+Rails.configuration.assets.paths.reject! do |path|
+  path.include?('rails-assets-bootstrap') && !path.include?('rails-assets-bootstrap-markdown')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.